src/panels/lovelace/cards/hui-calendar-card.ts

```diff
@@ -91,7 +89,7 @@ export class HuiCalendarCard extends LitElement implements LovelaceCard {
       };
     });
 
-    this._config = config;
+    this._config = { initial_view: "dayGridMonth", ...config };
```

Comment:
The default already is `dayGridMonth`, so this is not needed?

Reply:
Well, if `this._config.initialView` is `undefined` then we set it on the calendar as `undefined`. The property default gets overridden by `Undefined`. `ifDefined` directive is only for attributes I believe (or I didn't set it up correctly) so the default here is needed unless there is another way I have missed (which I am sure there is)